runtime-common/orocos-kdl/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=orocos-kdl
+PKGSEC=libs
+PKGDEP="catkin-pkg eigen-3 boost cppunit"
+PKGDES="Code for rigid body kinematics calculations and representations for kinematic structures and their inverse and forward kinematic solvers"
```

Comment:
Application-independent framework for modelling and computation of kinematic chains

https://www.orocos.org/kdl.html